domain/schema/model/sql/0022-resource.sql

```diff
@@ -31,6 +31,7 @@ CREATE TABLE resource (
     uuid TEXT NOT NULL PRIMARY KEY,
     application_uuid TEXT NOT NULL,
     name TEXT NOT NULL,
+    revision INT,
```

Comment:
**question:** How do we deal with overrides ? Like `juju deploy --resource my_resource=/path/to/file`

In this case, `juju resources` seems to yield a date as revision number. I assumes the idea here would be to get the resource, see that the revision is NULL, then print the creation date (`created_at` field). Sounds right ?

Reply:
@gfouillet, yes, that's why I didn't add `NOT NULL` to the revision.

I think using NULL will be more effective that -1 here. Because we didn't use a *int in the mongo docs often, it wasn't easy to determine if a int wasn't set. We now have a method to do that in sql.